### PR TITLE
Splash screen Added With Animation

### PR DESCRIPTION
**Pull request by advait2205**
As shown below a splash screen is added to the app consisting of app logo and app name
ezgif com-gif-maker

### CODE REVIEW

**Review by vidhibadrakiya — COMMENTED** [review]
Looks good.

**Review by vidhibadrakiya — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "com.adit.tictactoe"
-        minSdkVersion 15
```
Comment (adit19shah) [inline app/build.gradle:8]: Why have we changed the midSdkVersion ?

**Review by adit19shah — COMMENTED** [review]
Please rename the file SplashScreen to SplashScreenActivity and make necessary changes in other files.

File: app/build.gradle (hunk 1 above)
Comment (advait2205) [reply app/build.gradle:8]: The reason for change for minSdkversion is that creation of new activity requires a minimum SDK version in Android Studio or so i was notified. So,, had to change it.

Comment (adit19shah) [reply app/build.gradle:8]: Fine, we can keep it as 21 then.

**Comment by advait2205** [comment]
> Thanks @advait2205, have suggested few changes.
> 
> Please rename the file SplashScreen to SplashScreenActivity and make necessary changes in other files.

Done changed name to SplashScreenActivity and made changes in affected files

**Review by vidhibadrakiya — APPROVED** [review]

**Review by sarthakhp — APPROVED** [review]

**Comment by adit19shah** [comment]
LGTM, Thanks.

**Review by adit19shah — APPROVED** [review]

**Pull request merged**